src/fragments/lib/datastore/ios/advanced-workflows/custom-primary-key/delete-snippet.mdx

```diff
@@ -1 +1,72 @@
-## TODO (see js snippet for examples)
\ No newline at end of file
+
+A record with a custom primary key can be deleted in the following ways:
+
+With the value of the primary key:
+
+<BlockSwitcher>
+
+<Block name="Listener (iOS 11+)">
+
+```swift
+Amplify.DataStore.delete(Book.self, withIdentifier: .identifier(isbn: "12345"), where: nil) { result in
+    switch result{
+    case .success:
+        print("Book deleted!")
+    case .failure(let error):
+        print("Error deleting book with error \(error)")
+    }
+}
+
+```
+
+</Block>
+
+<Block name="Combine (iOS 13+)">
+
+```swift
+let sink = Amplify.DataStore.delete(Book.self, withIdentifier: .identifier(isbn: "12345")).sink {
+    if case let .failure(error) = $0 {
+        print("Error deleting book with error \(error)")
+    }
+}
+receiveValue: {
+    print("Book deleted!")
+}
+```
+</Block>
+
+</BlockSwitcher>
+
+With the value of Predicate:
+
+<BlockSwitcher>
+
+<Block name="Listener (iOS 11+)">
+
+```swift
+Amplify.DataStore.delete(Book.self, where: Book.keys.isbn == "12345" ) { result in
+    switch result {
+    case .success:
+        print("Book deleted!")
+    case .failure(let error):
+        print("Error deleting book - \(error)")
```

Comment:
let's pick message string and use it across all code snippets. this one looks good too. pick either
1. Error deleting book - \(error)
2. Error deleting book with error: \(error)